test/integration_test/applicationbackup_test.go

```diff
@@ -63,6 +64,10 @@ func TestApplicationBackup(t *testing.T) {
 }
 
 func TestScaleApplicationBackup(t *testing.T) {
+
+	if !defaultsBackupSet {
```

Comment:
This is to ensure that even if `TestScaleApplicationBackup` is run independently we set all the required defaults.
I noticed this during my testing when making these changes.